crates/bevy_ecs/src/change_detection.rs

```diff
@@ -28,15 +28,6 @@ pub const MAX_CHANGE_AGE: u32 = u32::MAX - (2 * CHECK_TICK_THRESHOLD - 1);
 /// ## Example
 /// Using types that implement [`DetectChanges`], such as [`ResMut`], provide
 /// a way to query if a value has been mutated in another system.
-/// Normally change detecting is triggered by either [`DerefMut`] or [`AsMut`], however
```

Comment:
I didn't think explaining how to bypass change detection on a type that can't cause change detection was the right location.

Reply:
Yeah, I agree there! But we should link to the subtrait in these docs: it's hard discover them otherwise.

I also think that mentioning that "change detection works via DerefMut" is important information.

Reply:
I think there is a comment pointing towards DetectChangesMut now to point that (I used ResMut as an example too)